windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,75 @@
+
+[IMPORTANT]
+====
+Running Windows container workloads is not supported for clusters in a restricted network or disconnected environment.
+====
+
+Version 3.x of the WMCO is only compatible with {product-title} 4.8.
+
+[id="wmco-3-0-0-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+// Addresses Github #32620 - https://github.com/openshift/openshift-docs/pull/32620
+==== Clarified limits on custom VXLAN port selection
+
+Users may not select a custom VXLAN port when using the latest version of Windows server.
+
+// Addresses WINC-551 - https://github.com/openshift/windows-machine-config-operator/pull/279
+==== Added support for autoscaling Windows nodes
+
+In this release, Windows nodes now support cluster autoscaling. Cluster autoscaling automatically scales up or down the number of machines cluster to meet its current deployment needs, keeping workloads available and pruning unused resources when possible.
+
+[id="wmco-3-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, The load balancer service would become unstable when you would scale additional Windows nodes. Now, the load balancer service has been stabilized to allow for adding multiple Windows nodes without erratic performance. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[**BZ#1905950**])
+
+* (link:https://bugzilla.redhat.com/show_bug.cgi?id=1919391[**BZ#1919391**])
+
+* (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924559[**BZ#1924559**])
+
+* Previously, when you replaced a cloud-private-key secret with a new private key, the WMCO patched pub-key-has with an annotation to the Linux node as well as the master and worker node. Now, when you change this private key, no annotation is added. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930791[**BZ#1930791**])
+
+* Previously, the kube-proxy service would terminate unexpectedly after the load balancer is created if you created the load balancer after the Windows pods begin running. Now, the kube-proxy service does not crash when recreating the load balancer service. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[**BZ#1939968**])
+
+* Previously, the `windows_exporter` component reported various metrics as `windows_*`. This error caused some of the node-level metrics that provide insight into the nodes through telemetry services to go unreported. Now, the componets export correctly showing all of the expected metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1948037[**BZ#1948037**])
+
+* Previously, empty IP address values in the load balancer's ingress would break the data path and the Windows service would be unreachable. This issue was due to the kube-proxy component considering empty IP address values in the ingress as invalid. Now, the Windows service is reachable even if the IP address value is empty. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**])
```

Comment:
[This comment](https://github.com/openshift/openshift-docs/pull/34130#discussion_r661766906) applies here too